config_test.go

```diff
@@ -76,6 +75,5 @@ func TestAltDNSNamesWithArg(t *testing.T) {
 // init parses the -test.* flags from the command line arguments list and then
 // removes them to allow go-flags tests to succeed.
 func init() {
-	flag.Parse()
```

Comment:
do we know why this was added and why tests are still passing without it? The function doesn't appear to do what the comment says anymore without it. (I know why this has to be removed, but i'm wondering if we should put it in a TestMain instead)

Reply:
Many moons ago the test flags ended up getting passed along to go-flags if they weren't consumed by parsing them.  That seems to have changed at some point as I tested go1.11, go1.12, and go1.13 without failure.  I didn't go back further than go.11 since it gets pretty hard to build before modules.